[2022-08-11 23:09:30,058] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2015 manual__2022-08-11T23:09:24.436624+00:00 [queued]>
[2022-08-11 23:09:30,073] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2015 manual__2022-08-11T23:09:24.436624+00:00 [queued]>
[2022-08-11 23:09:30,074] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:09:30,079] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-11 23:09:30,081] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:09:30,096] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Ingest_from_2015> on 2022-08-11 23:09:24.436624+00:00
[2022-08-11 23:09:30,114] {standard_task_runner.py:52} INFO - Started process 6215 to run task
[2022-08-11 23:09:30,131] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG_US_FOOD_NUTRITION', 'Ingest_from_2015', 'manual__2022-08-11T23:09:24.436624+00:00', '--job-id', '716', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp_na4gica', '--error-file', '/tmp/tmph4qicc7z']
[2022-08-11 23:09:30,136] {standard_task_runner.py:80} INFO - Job 716: Subtask Ingest_from_2015
[2022-08-11 23:09:30,280] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG_US_FOOD_NUTRITION.Ingest_from_2015 manual__2022-08-11T23:09:24.436624+00:00 [running]> on host a02c2b8cab8b
[2022-08-11 23:09:30,364] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG_US_FOOD_NUTRITION
AIRFLOW_CTX_TASK_ID=Ingest_from_2015
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T23:09:24.436624+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T23:09:24.436624+00:00
[2022-08-11 23:09:38,249] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table1_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:39,579] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table2_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:40,777] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table3_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:42,433] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table4_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:43,793] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table5_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:45,134] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table6_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:46,402] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table7_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:48,091] {logging_mixin.py:115} INFO - Successfully uploaded 2015/Table8_2015.xlsx to s3-bucket-raw-usda
[2022-08-11 23:09:48,093] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 23:09:48,103] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG_US_FOOD_NUTRITION, task_id=Ingest_from_2015, execution_date=20220811T230924, start_date=20220811T230930, end_date=20220811T230948
[2022-08-11 23:09:48,129] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 23:09:48,160] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
